modules/digitaluniverse/rendering/renderableplanescloud.cpp

```diff
@@ -524,15 +524,17 @@ void RenderablePlanesCloud::initializeGL() {
 }
 
 
-void RenderablePlanesCloud::deleteDataGPU() {
-    for (RenderingPlane& renderingPlane : _renderingPlanesArray) {
-        glDeleteVertexArrays(1, &renderingPlane.vao);
-        glDeleteBuffers(1, &renderingPlane.vbo);
+void RenderablePlanesCloud::deleteDataGPUAndCPU() {
+    for (std::pair<int, PlaneAggregate> pAMapItem : _planesMap) {
```

Comment:
Needs to be a reference or the clear on line 531 will clear the copy

Reply:
You are write. My bad. :-)
Fixed.